src/main/java/org/apache/commons/collections4/CollectionUtils.java

```diff
@@ -1266,9 +1248,6 @@ public static Object get(final Object object, final int index) {
         } else if (object instanceof Iterable<?>) {
             final Iterable<?> iterable = (Iterable<?>) object;
             return IterableUtils.get(iterable, i);
-        } else if (object instanceof Collection<?>) {
```

Comment:
removing of this condition is safe .moreover, this check is useless, because, any collection implements interface Iterable, and if object is Collection, then previous condition `(object instanceof Iterable<?>)` is true.